Util/Xamarin.AndroidBinderator/Xamarin.AndroidBinderator.Tool/Xamarin.AndroidBinderator.Tool.csproj

```diff
@@ -12,7 +12,7 @@
 
   <PropertyGroup>
     <PackageId>Xamarin.AndroidBinderator.Tool</PackageId>
-    <PackageVersion>0.5.3</PackageVersion>
+    <PackageVersion>0.5.4</PackageVersion>
```

Comment:
OK I need to be careful - my PR has the same version

https://github.com/xamarin/XamarinComponents/pull/1367

both can be merged and published together